### DISCUSSION TITLE
High GPU usage.

### DISCUSSION THREAD

**Discussion by Ianlucht** [OpenTabletDriver/OpenTabletDriver#1467] [Q&A] [answered]
So I have noticed that there is a noticable amount of GPU usage by OTD. This has caused input delay in osu. I wonder if and how to fix this problem.
![OTDusage](https://user-images.githubusercontent.com/90893791/134794715-56796afc-03a0-4c9b-b285-c3199a59404a.PNG)
![otdarea](https://user-images.githubusercontent.com/90893791/134794717-a96347eb-339b-40b4-ae7d-4183728be568.PNG)
.

**Comment by X9VoiD** [comment, marked as answer]
This is not OTD's fault (the issue stems from Eto framework and Window's slow WPF rendering), and we already implemented "fixes" by stopping rendering when OTD is minimized.
